src/example-relay/src/graphql/services/connectionFromArray.js

```diff
@@ -6,18 +6,17 @@ import {
   offsetToCursor,
   type Connection,
   type ConnectionArguments,
-} from 'graphql-relay';
+} from '@adeira/graphql-relay';
 
-type Args = $ReadOnly<{
+type Args = {
   ...ConnectionArguments,
   ...
-}>;
+};
 // There is open discusion how to handle bidirectional pagination:
 // https://github.com/graphql/graphql-relay-js/issues/58
 //
 // Until Relay devs decide, let's try our implementation
 export default function connectionFromArray<T>(data: $ReadOnlyArray<T>, args: Args): Connection<T> {
```

Comment:
I believe we should completely get rid of this function now, right? Can you please create an issue?

Reply:
Yes, that is the plan, I can create the task 👍